apps/marketplace/actions/briefActions.js

```diff
@@ -34,21 +27,16 @@ export const loadBrief = briefId => dispatch => {
 
 export const handleBriefResponseSuccess = () => ({ type: BRIEF_RESPONSE_SUCCESS })
 
-export const handleBriefResponseFailure = response => {
+export const handleBriefResponseFailure = response => dispatch => {
```

Comment:
don't put a `dispatch` here! dispatch belongs in containers. this is stinky

Reply:
I think it's fine to put thunk async actions here. Primary actions in containers 👍

Reply:
Excuse my naivety @elisechant, but doesn't @gaearon himself say this is ok https://github.com/gaearon/redux-thunk#composition ?